src/assets/css/app.css

```diff
@@ -2898,7 +2898,7 @@ textarea.form-control-lg {
   height: 0.5rem;
   color: transparent;
   cursor: pointer;
-  background-color: var(--bs-tertiary-bg);
+  background-color: #1778f2;
```

Comment:
1. Should this be moved to `style/app.module.css` for the components? 
2. Is the Left Nav slider?